pages/api/revalidate.js

```diff
@@ -23,6 +31,9 @@ async function handler(req, res) {
   }
 
   try {
+    const searchParams = new URLSearchParams(query);
+
+    await fetch(`${BASE_URL}/api/app-revalidate?${searchParams.toString()}`);
```

Comment:
forward the request to `/app-revalidate`. Eventually this API will be deprecated and `/app-revalidate` will be renamed to `/revalidate` and assume the role as primary revalidator